dd-sdk-android-core/build.gradle.kts

```diff
@@ -170,9 +170,11 @@ dependencies {
     testImplementation(libs.bundles.testTools)
     unmock(libs.robolectric)
 
-    // Static Analysis
-    // TODO MTG-12 detekt(project(":tools:detekt"))
-    // TODO MTG-12 detekt(libs.detektCli)
+    // Test Fixtures
+    testFixturesImplementation(libs.kotlin)
```

Comment:
why it is needed? `implementation` dependencies don't cover `testFixtures` source set (like they cover `test` source set)?

Reply:
For Gradle, no it's not necessary but AndroidStudio needed that to be happy.